core/migrate/data.go

```diff
@@ -97,9 +97,9 @@ var migrations = []migration{
 		ALTER TABLE config ALTER COLUMN blockchain_id SET DATA TYPE bytea USING decode(blockchain_id, 'hex');
 	`},
 	{Name: "2017-01-13.0.core.asset-definition-bytea.sql", SQL: `
-		ALTER TABLE assets 
+		ALTER TABLE assets
 			ALTER COLUMN definition SET DATA TYPE text;
-		ALTER TABLE assets 
+		ALTER TABLE assets
```

Comment:
I don't think we can modify and of these older migrations. It'll change the SHA. If we really don't care about preserving data since it requires a blockchain reset anyways, we should just collapse all the migrations into a new single migration. We can do that in this PR or a new one.